bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -580,23 +580,16 @@ private void create_24_Tour(final StringBuilder sb) {
 
       if (isDescription || isTitle || isWeather || isTourTags) {
 
-         sb.append("<div class='action-hover-container' style='margin-top:30px; margin-bottom: 5px;'>" + NL); //$NON-NLS-1$
+         sb.append("<div class='action-hover-container' style='margin-top:15px; margin-bottom: 5px;'>" + NL); //$NON-NLS-1$
          {
 
             sb.append("<div class='blog-item'>"); //$NON-NLS-1$
             {
                /*
                 * Tour title
                 */
-               if (isTitle == false) {
-
-                  tourTitle = "&nbsp;"; //$NON-NLS-1$
-
-               } else {
-
-                  if (UI.IS_SCRAMBLE_DATA) {
-                     tourTitle = UI.scrambleText(tourTitle);
-                  }
+               if (UI.IS_SCRAMBLE_DATA) {
+                  tourTitle = UI.scrambleText(tourTitle);
                }
```

Comment:
This part could be move to the location in the code, where the title is applied

Reply:
You mean on the line below ?
`final String hoverEdit = WEB.escapeSingleQuote(NLS.bind(Messages.Tour_Blog_Action_EditTour_Tooltip, tourTitle));`

Reply:
I thought it could be moved into `if (isTitle...` but the title was already used before, so ignore my previous comment